docs/guides/jupyter.rst

```diff
@@ -0,0 +1,293 @@
+  at the time of writing.
+
+.. _ipywidgets: https://ipywidgets.readthedocs.io/
+
+.. figure:: /_static/images/guides/jupyterlab-save-widget-state.png
+   :width: 30%
+   :align: center
+   :alt: JupyterLab option to save the interactive widget state automatically
+
+   JupyterLab option to save the interactive widget state automatically
+
+For example, if you create a notebook with a simple
+`IntSlider <https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20List.html#IntSlider>`_
+widget from ipywidgets and save the widget state,
+the slider will render correctly in Sphinx.
+
+.. figure:: /_static/images/guides/widget-html.gif
+   :width: 80%
+   :align: center
+   :alt: Interactive widget rendered in HTML by Sphinx
+
+   Interactive widget rendered in HTML by Sphinx
+
+To see more elaborate examples:
+
+- `ipyleaflet`_ provides several widgets for interactive maps,
+  and renders live versions of them `in their
+  documentation <https://ipyleaflet.readthedocs.io/en/latest/api_reference/velocity.html>`_.
+- `PyVista <https://docs.pyvista.org/>`_ is used for scientific 3D visualization
+  with several interactive backends and `examples in their
+  documentation <https://docs.pyvista.org/index.html#maps-and-geoscience>`_ as well.
+
+.. warning::
+
+   Although widgets themselves can be embedded in HTML as shown above,
+   `events <https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20Events.html>`_
+   require a backend (kernel) to execute.
+   Therefore, ``@interact``, ``.observe``, and related functionalities relying on them
+   will not work as expected.
+
+Creating galleries of examples using notebooks
+----------------------------------------------
+
+If you want to create an HTML gallery of examples for your project,
+`Sphinx-Gallery <https://sphinx-gallery.github.io/>`_
+offers a very convenient way of doing so from Sphinx,
+including thumbnail generation.
+However, `it works with Python scripts rather than
+notebooks <https://github.com/sphinx-gallery/sphinx-gallery/issues/245>`_,
+which might not be what you want.
+
+`nbsphinx`_ has support for `creating thumbnail galleries from a list of Jupyter
+notebooks <https://nbsphinx.readthedocs.io/en/latest/subdir/gallery.html>`_,
+and it is compatible with Sphinx-Gallery styles.
+To use it, you will need to install both nbsphinx and Sphinx-Gallery,
+and modify your ``conf.py`` as follows:
+
+.. code-block:: python
+   :caption: conf.py
+
+   extensions = [
+      'nbsphinx',
+      'sphinx_gallery.load_style',
+   ]
+
+After doing that, there are two ways to create the gallery:
+
+- From a reStructuredText source file, using the ``.. nbgallery::`` directive,
+  `as showcased in the
+  documentation <https://nbsphinx.readthedocs.io/en/latest/a-normal-rst-file.html#thumbnail-galleries>`_.
+- From a Jupyter notebook, adding a ``"nbsphinx-gallery"`` tag to the metadata of a cell.
+  Each editor has a different way of modifying the cell metadata (see figure below).
```

Comment:
Do we want to recommend this approach? Seems like the RST way of doing it is way more obvious and nicer -- and doesn't require us talking about editors.

Reply:
Yes! In fact, it's done in the next paragraph. Perhaps I can rephrase this to clarify that?

Reply:
Given that we are doing that, resolving this comment